io/io/src/TGenCollectionProxy.cxx

```diff
@@ -1043,6 +1043,7 @@ void* TGenCollectionProxy::At(UInt_t idx)
    if ( fEnv && fEnv->fObject ) {
       switch (fSTL_type) {
       case ROOT::kSTLvector:
+      case ROOT::kROOTRVec:  // TODO will be unnecessary with RVec 2.0
```

Comment:
Can you expand the comment saying "why" it wont be necessary.

Reply:
done